default_dial_option_server_option_test.go

```diff
@@ -74,6 +75,34 @@ func (s) TestDisableGlobalOptions(t *testing.T) {
 	internal.ClearGlobalDialOptions()
 }
 
+type testPerTargetDialOption struct{}
+
+func (do *testPerTargetDialOption) DialOption(parsedTarget url.URL) DialOption {
+	if parsedTarget.Scheme == "passthrough" {
+		return WithTransportCredentials(insecure.NewCredentials()) // credentials provided, should pass NewClient.
+	}
+	return EmptyDialOption{} // no credentials, should fail NewClient
+}
+
+// TestGlobalPerTargetDialOption configures a global per target dial option that
+// produces transport credentials for channels using "passthrough" scheme.
+// Channels that use the passthrough scheme should this be successfully created
```

Comment:
"This" makes no sense.  :laughing: 

(Delete "this")

Reply:
Whoops haha, deleted.